NUAuditableEntity.js

```diff
@@ -0,0 +1,45 @@
+import NUAttribute from './NUAttribute';
+import NUEntity from './NUEntity';
+
+/*
+  This class models AuditableEntity
+*/
+export default class NUAuditableEntity extends NUEntity {
+    static attributeDescriptors = {
+        ...NUEntity.attributeDescriptors,
+        creationDate: new NUAttribute({
+            localName: 'creationDate',
+            attributeType: NUAttribute.ATTR_TYPE_TIMESTAMP,
+            isReadOnly: true,
```

Comment:
this should be creationOnly right?

Reply:
@natabal UI will not be sending values for these attributes in POST. Server auto-assigns these values. So I think it should stay as isReadOnly

Reply:
@killanch For correctness you need to add creationOnly as well.